src/i18n/locales/en/translation.json

```diff
@@ -1169,22 +1245,36 @@
   },
   "permissions": {
     "webln": {
-      "getbalance": "Read the balance of your account"
+      "getbalance": {
+        "title": "Get Balance",
+        "description": "Read the balance of your account"
+      }
     },
     "bitcoin": {
-      "getaddress": "Read your Bitcoin receive address"
+      "getaddress": {
+        "title": "Get Address",
```

Comment:
here you have the first word uppercase - for nostr it's all lowercase

Reply:
resolved